presto-tests/src/test/java/com/facebook/presto/tests/TestTpchTaskUpdateSizeLimitIntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.Session;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.tpch.TpchQueryRunner.createQueryRunner;
+
+public class TestTpchTaskUpdateSizeLimitIntegrationTest
+        extends AbstractTestQueryFramework
+{
+    @SuppressWarnings("unused")
+    public TestTpchTaskUpdateSizeLimitIntegrationTest()
+    {
+        this(() -> createQueryRunner(ImmutableMap.of("experimental.internal-communication.max-task-update-size", "10kB")));
+    }
+
+    protected TestTpchTaskUpdateSizeLimitIntegrationTest(QueryRunnerSupplier queryRunnerSupplier)
+    {
+        super(queryRunnerSupplier);
+    }
+
+    @Test
+    public void testTaskUpdateSizeLimit()
```

Comment:
The test does not pass locally. Can we make something more reliable?

Reply:
The test passes locally when i tested on my laptop. Not sure why it fails here. Do you have any suggestion as to how I should write a query so that it bloats up the TaskUpdate size and fails??

Reply:
Reduce the allowed size to a very low number? What if this is reduced to `100B` ?

Reply:
Reducing the allowed size to 100B fixed it.